docs/source/getting-started.mdx

```diff
@@ -10,23 +10,18 @@ This tutorial helps you:
 * Run an instance of Apollo Server that lets you execute queries against your schema
 
 This tutorial assumes that you are familiar with the command line and
-TypeScript or JavaScript, and that you have a recent version of Node.js (14+) installed.
+JavaScript and have installed a recent Node.js (14+) version. Additionally, for those interested, this tutorial includes an optional section describing how to set up Apollo Server with TypeScript.
```

Comment:
I feel like this isn't quite aligned with our goal of TypeScript everywhere. Do we want a vanilla JS example at all?

We can bump the node version here to 16+ as well (which is the current LTS version).

Reply:
You bring up a really good point, I actually originally wrote this "Getting Started" guide to only include TS.

However, Stephen brought up the point that we should try to cater to both TS & JS folks where we can, because why not try to reach as many developers as possible.  Especially when the AS3 docs are *so* JS-focused, having AS4 not be inclusive of those using JS users might turn people away from migrating. Totally open to further discussion on this though!

Reply:
I think that's a good perspective. With that in mind, I think the direction forward would be to write everything assuming TypeScript, with collapsible bits for "if you insist on using JS". Does that sound reasonable?

Most importantly, I think we want our format to convey the message that TS is strongly preferred but we won't leave you in the dark if you're using JS.